src/schema/v2/__tests__/tag.test.js

```diff
@@ -2,7 +2,8 @@
 import { runQuery } from "schema/v2/test/utils"
 
 describe("Tag", () => {
-  describe("For just querying the tag artworks", () => {
+  // FIXME: Tag no longer has a direct connection to filtered artworks
+  describe.skip("For just querying the tag artworks", () => {
```

Comment:
Yup, this was just entirely removed for now, as Emission didn’t need it, and so we should either add it back when we migrate Reaction or just leave it out forever.